sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubsRetryOptions.cs

```diff
@@ -91,10 +91,10 @@ public TimeSpan TryTimeout
             {
                 if (value < TimeSpan.Zero)
                 {
-                    throw new ArgumentException(Resources.TimeoutMustBePositive, nameof(TryTimeout));
+                    throw new ArgumentException(Resources.TimeoutMustBePositive, nameof(value));
```

Comment:
I'd like to preserve the property name here with the justification that `value` is an implicit name that isn't obvious to callers.  The name of the property would seem to be more meaningful in this context.

Reply:
SGTM.  I'll suppress these.  To be honest, I had to search for "what is the correct name of the parameter to use here", so I'm happy to keep things as is.